langgraph_bot/insert_bot_data.py

```
@@ -0,0 +1,36 @@
+                "content": {
+                    "summary": post.get("summary"),
+                    "description": post.get("description")
+                },
+                "isoldpost": False
+            }
+
+            supabase.table("post_content").insert(content_data).execute()
+
+            print(f"Inserted: {post['title']}")
+
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)